content/en/docs/instrumentation/java/exporters.md

```diff
@@ -45,6 +50,46 @@ dependencies {
 
 {{< /tab >}} {{< /tabpane>}}
 
+Under the hood, there are two protocol options supported, each with different
+"sender" implementations.
+
+- `grpc` - gRPC implementation of OTLP exporters, represented by
+  `OtlpGrpcSpanExporter`, `OtlpGrpcMetricExporter`, `OtlpGrpcLogRecordExporter`.
+- `http/protobuf` - HTTP with protobuf encoded payload implementation of OTLP
+  exporters, represented by `OtlpHttpSpanExporter`, `OtlpHttpMetricExporter`,
+  `OtlpHttpLogRecordExporter`.
+
+A sender is an abstraction which allows different gRPC / HTTP client
+implementations to fulfill the OTLP contract. Regardless of the sender
+implementation, the same exporter classes are used. A sender implementations is
+automatically used when it is detected on the classpath. The sender
+implementations are described in detail below:
+
+- `{groupId}:{artifactId}` - Sender description.
+- `io.opentelemetry:opentelemetry-exporter-sender-okhttp` - The default sender,
+  included automatically with `opentelemetry-exporter-otlp` and bundled with the
+  OpenTelemetry Java agent. This includes an
+  [OkHttp](https://square.github.io/okhttp/) based implementation for both the
+  `grpc` and `http/protobuf` versions of the protocol, and will be suitable for
+  most users. However, OkHttp has a transitive dependency on kotlin which is
+  problematic in some environments.
+- `io.opentelemetry:opentelemetry-exporter-sender-jdk` - This sender includes a
+  JDK 11+
+  [HttpClient](https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpClient.html)
+  based implementation for the `http/protobuf` version of the protocol. It
+  requires zero additional dependencies, but requires java 11+. To use, include
+  the artifact and explicitly exclude the default
+  `io.opentelemetry:opentelemetry-exporter-sender-okhttp` dependency.
+- `io.opentelemetry:opentelemetry-exporter-sender-grpc-managed-channel` - This
+  sender includes a [grpc-java](https://github.com/grpc/grpc-java) based
+  implementation for the `grpc` version of the protocol. To use, include the
+  artifact, explicitly exclude the default
+  `io.opentelemetry:opentelemetry-exporter-sender-okhttp` dependency, and
+  include one of the
+  [grpc transport implementations](https://github.com/grpc/grpc-java#transport).
+
```

Comment:
I think what's missing is an explanation of when I'd want to use a different sender implementation than the default.

Reply:
Gotcha. I suppose most java developers who are in this situation will recognize when they need to swap a sender implementation.